src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Fruit> fruits = new HashMap<>();
```

Comment:
According to the encapsulation principle, class fields should be private. Exposing the `fruits` map as public breaks encapsulation. Consider changing the access modifier to private and provide public methods to interact with the storage.